docs/plugins/plugin-script.asciidoc

```diff
@@ -188,7 +188,7 @@ purge the configuration files while removing a plugin, use `-p` or `--purge`.
 This can option can be used after a plugin is removed to remove any lingering
 configuration files.
 
-[[removing-multiple-plugins]]
+[discrete]
```

Comment:
```suggestion
[discrete]
[[removing-multiple-plugins]]
```

Reply:
I wouldn't remove the ID. It helps us link to this section directly from other places in the docs.

Reply:
Ah ok, got it. Thanks for catching that. I'll commit your suggestion.